contentcuration/contentcuration/frontend/channelEdit/views/files/FileUpload.vue

```diff
@@ -134,6 +131,10 @@
           'order'
         );
       },
+      selectedFilename() {
+        const file = this.files.find(f => f.id === this.selected);
+        return file ? file.original_filename : '';
```

Comment:
Any opinion of not using use nullish coalescing here?

Reply:
I think using nullish coalescing(`??`) would also require the use of optional chaining(`?`). I think we would be looking at something like
`file?.original_filename ?? ''`?

Im not strongly opinionated on the use of the `??` or `?`, although I think they can be less explicit in some cases.